src/traces/scattermapbox/attributes.js

```diff
@@ -26,6 +26,58 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            role: 'info',
+            dflt: false,
+            editType: 'plot',
+            description: 'Determines whether clustering is enabled or disabled.'
+        },
+        maxzoom: {
```

Comment:
We could possibly drop this attribute and instead reuse `layout.mapbox.(min|max)zoom`, right?
https://github.com/plotly/plotly.js/blob/39fe198ad844e62432d30e11b862b34ef9b1f193/src/plots/mapbox/layout_attributes.js#L241-L251
If not, we should probably make `max`, `dflt` and `description` values similar to those of `mapbox.maxzoom`.

Reply:
That's `layout.mapbox.layers[i].(min|max)zoom` and determines the visibility of just that layer. This has a similar effect, but only for this one trace, so seems necessary to me.

Curious though why they have different `max` values (22 vs 24)?